src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -361,15 +367,46 @@
            workspace-attributes (get-in props [:workspace :workspace :workspace-attributes])]
        (build-autocomplete-list
         (utils/restructure workspace-attributes entity-types selected-entity-type))))
-   :-load-validated-method-config
+   :-load-method-config
    (fn [{:keys [state props this]}]
+     (endpoints/call-ajax-orch
+      {:endpoint (endpoints/get-workspace-method-config (:workspace-id props) (:config-id props))
+       :on-done (fn [{:keys [success? get-parsed-response status-text]}]
+                  (if success?
+                    (this :-load-inputs-outputs (get-parsed-response))
+                    (swap! state assoc :error status-text)))}))
+   :-load-inputs-outputs
+   (fn [{:keys [state this]} unvalidated-mc]
+     (endpoints/call-ajax-orch
+      {:endpoint endpoints/get-inputs-outputs
+       :payload (:methodRepoMethod unvalidated-mc)
+       :headers ajax/content-type=json
+       :on-done (fn [{:keys [success? get-parsed-response]}]
+                  (if success?
+                    (this :-load-validated-method-config (get-parsed-response))
+                    (let [fake-inputs-outputs (fn [data]
+                                                (let [method-config (:methodConfiguration data)]
```

Comment:
Just have this function take `unvalidated-mc` so you don't need to unpack it again